gradle/java.gradle

```diff
@@ -297,6 +224,12 @@ tasks.withType(Test).configureEach {
   testLogging {
     exceptionFormat = 'full'
   }
+
+  if (!isJavaVersionAllowed(JavaVersion.toVersion(11))) {
```

Comment:
Can you add comment explaining this?

Reply:
Yeah - also made sure it only applies to test task which I guess is more appropriate.